connectors/iotf/src/main/java/quarks/connectors/iotf/runtime/IotfDeviceCommands.java

```diff
@@ -41,8 +45,7 @@ public void accept(Consumer<Command> commandSubmitter) {
         try {
             connector.subscribeCommands(commandSubmitter);
         } catch (Exception e) {
-            // TODO Auto-generated catch block
-            e.printStackTrace();
+			logger.error("Exception caught while subscribing commands: {}", e);
```

Comment:
Coding conventions are missing from DEVELOPMENT.md but that effort is underway (e.g., QUARKS-23).
For the most part the code uses spaces instead of hard tabs, with 4 spaces of indentation.  (the above may be changing that to 2).  Please remove the hard tabs.

The ": {}" in the error() msg string needs to be removed here and in other cases below.
I noticed that the sample use that I pointed you at in JobRegistry had that problem (I notified the author).  Sorry about that.

Thanks for your help!